orc8r/cloud/go/obsidian/swagger/handlers/handlers.go

```diff
@@ -14,17 +14,50 @@
 package handlers
 
 import (
+	"bytes"
+	"html/template"
 	"net/http"
+	"strings"
 
 	"magma/orc8r/cloud/go/obsidian"
 	"magma/orc8r/cloud/go/obsidian/swagger"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/lib/go/registry"
 
 	"github.com/labstack/echo"
+	"github.com/labstack/echo/middleware"
+	"github.com/pkg/errors"
 )
 
-// GetGenerateCombinedSpecHandler returns a routing handler which creates
+// RegisterSwaggerHandlers registers routes for Swagger specs and
+// the Swagger UI.
+func RegisterSwaggerHandlers(e *echo.Echo) error {
+	trailSlashMiddleware := middleware.AddTrailingSlashWithConfig(middleware.TrailingSlashConfig{
+		RedirectCode: http.StatusMovedPermanently,
+	})
+
+	err := registerSwaggerSpecHandlers(e, trailSlashMiddleware)
+	if err != nil {
+		return obsidian.HttpError(err, http.StatusInternalServerError)
```

Comment:
No need to return http errors from this function, since the errors aren't being propagated back to an http client

Reply:
Gotchu, thanks for the catch